test/system_blackbox.py

```diff
@@ -64,26 +66,31 @@ def setUpClass(cls):
 
     def tearDown(self):
         # Reset all topics.
-        t = twist(0, 0)
-        l = Bool(False)
+        twist_msg = twist(0, 0)
+        unlock = Bool(False)
 
-        self._vel1.pub(t)
-        self._vel2.pub(t)
-        self._vel3.pub(t)
+        self._vel1.pub(twist_msg)
+        self._vel2.pub(twist_msg)
+        self._vel3.pub(twist_msg)
 
-        self._lock1.pub(l)
-        self._lock2.pub(l)
+        self._lock1.pub(unlock)
+        self._lock2.pub(unlock)
 
         # Wait for previously published messages to time out,
         # since we aren't restarting twist_mux.
         #
         # This sleeping time must be higher than any of the
         # timeouts in system_test_config.yaml.
-        rospy.sleep(self.MESSAGE_TIMEOUT + self.TOPIC_TIMEOUT)
+        #
+        # TODO(artivis) use rate once available
```

Comment:
These in-code TODO-s will be forgotten and never addressed. Could you please instead open an issue on github with some detail and put `TODO: link....` here?